ledfx/effects/audio.py

```diff
@@ -196,6 +197,12 @@ def activate(self):
             )
             device_idx = default_device
 
+        if device_idx == -1:
```

Comment:
Drop a comment here outlining why we have a device idx of -1

Reply:
Thats implied by the warning text though, surely